### PR TITLE
Feat/412/optional user option

### LINKED ISSUE #412: [FEAT] - Optional User Parameter
### What feature would you like to see?

The `/faq` command should be given a `user` option. This needs to be **not required**.

If the command is passed a value for this parameter, the faq response should be modified to include a `content` value of `Hey <@!${user.id}, this should answer your question`.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by danicunhac**
# Pull Request

## Description:
Adds optional user option in the `faq` command.
Adds `ts-node-dev` for better development experience.

## Related Issue:

Closes #412

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "prebuild": "rm -rf ./prod",
     "build": "tsc",
     "start": "node -r dotenv/config ./prod/index.js",
+    "dev": "tsnd -r dotenv/config 'src/index.ts' --watch",
```
Comment (naomi-lgbt) [inline package.json:10]: We should not add dependencies in this PR - it's out of scope.

Additionally, Discord bots have login rate limits and watch mode is not ideal for them.

File: src/commands/faq.ts [hunk 2]
```diff
@@ -28,9 +33,15 @@ export const faq: Command = {
...
+      const userId = interaction.options.getString("user");
+
+      const description = userId
+        ? `Hey ${userId}, this should answer your question.\n${target.content}`
+        : target.content;
+
       const embed = new EmbedBuilder();
       embed.setTitle(target.title);
-      embed.setDescription(target.content);
+      embed.setDescription(description);
```
Comment (naomi-lgbt) [inline src/commands/faq.ts:44]: We should not add this conditional string to the embed description (it won't ping the user).

Instead, it should be passed as the `content` property in the `interaction.editReply` object below.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (danicunhac) [reply package.json:10]: Do they have rate limits even during development?

Comment (naomi-lgbt) [reply package.json:10]: Discord does not distinguish between development and production. A bot is a bot.

**Review by naomi-lgbt — APPROVED** [review]
banger pr, thanks for contributing!

**Comment by naomi-lgbt** [comment]
## Congratulations

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labelled `help wanted` for additional opportunities to contribute.

You can also see all open issues [through our contributor tool](https://contribute.naomi.lgbt)

**Pull request merged**